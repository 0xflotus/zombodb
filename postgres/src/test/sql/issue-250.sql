select id from so_Posts where zdb('so_posts', ctid) ==> 'id:~"^.*"';