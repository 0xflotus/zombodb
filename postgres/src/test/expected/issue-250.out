select id from so_Posts where zdb('so_posts', ctid) ==> 'id:~"^.*"';
ERROR:  {
  "_shards" : {
    "total" : 0,
    "successful" : 0,
    "skipped" : 0,
    "failed" : 5,
    "failures" : [
      {
        "shard" : 0,
        "index" : "contrib_regression.public.so_posts.idxso_posts",
        "status" : "BAD_REQUEST",
        "reason" : {
          "type" : "query_shard_exception",
          "reason" : "Can only use regexp queries on keyword and text fields - not on [id] which is of type [long]",
          "index_uuid" : "XjiwNU13REauOQgGRcpSzQ",
          "index" : "contrib_regression.public.so_posts.idxso_posts"
        }
      }
    ]
  }
}
